scenarioo-client/app/app.js

```diff
@@ -16,26 +16,10 @@
  */
 
 
-require('../node_modules/bootstrap/dist/css/bootstrap.min.css');
-require('./components/font-awesome/css/font-awesome.min.css');
+require('./vendor.js');
```

Comment:
I think the name `vendor` is a little bit strange here. I think most important global dependencies on 3rd party libraries could have stayed in the app.js